crates/cli/src/bin/wasm-bindgen-test-runner/main.rs

```diff
@@ -44,6 +44,11 @@ fn main() -> anyhow::Result<()> {
         None => bail!("must have a file to test as first argument"),
     };
 
+    let wasm_file_to_test_name = wasm_file_to_test
+        .file_name()
+        .and_then(|s| s.to_str())
+        .ok_or_else(|| anyhow!("file to test is not a valid file, can't extract file name"))?;
```

Comment:
```suggestion
        .context("file to test is not a valid file, can't extract file name")?;
```
That should work, see https://docs.rs/anyhow/latest/anyhow/trait.Context.html#tymethod.context.